### DISCUSSION TITLE
Importing into Platform I/O?

### DISCUSSION THREAD

**Discussion by Jediweirdo** [ukw100/IRMP#98] [General]
I want to use this library in VSCode via Platform IO and I see that it used to be available there, but the most up-to-date build on Platform I/O's registry is 2 years old. Anyway, is there a way to just manually import it without having to deal with all these include errors? Been trying for hours...

**Comment by ArminJo** [comment]
Below my mail to PlatformIO support:

Hello,

the actively maintained IRMP library is this: https://registry.platformio.org/libraries/arminjo/IRMP
and publishing is done by release management.

Please delete the old https://registry.platformio.org/libraries/ukw100/IRMP library entry
because it leads to unnecessary confusions and time wasting of users - see below.

Thank you very much!
Sincerely
Armin Joachimsmeyer - Maintainer of https://github.com/IRMP-org/IRMP
